debian/changelog

```diff
@@ -1,3 +1,9 @@
+raspberrypi-firmware (1:9.20230425+1-5.10.152-2+revpi10+1) buster; urgency=medium
```

Comment:
This will break the upgrade path, if we do not make a new debian revision for bullseye. '1:9.20230425+1-5.10.152-2+revpi10+1' is greater than '1:9.20230425+1-5.10.152-1+revpi11+1' - We have to increase the last 1 like this '1:9.20230425+1-5.10.152-1+revpi10+2'